torch/csrc/jit/passes/quantization.cpp

```diff
@@ -1876,7 +1861,11 @@ ModuleMethodVector InsertQuantDeQuantHelper::getInvokedMethods(
         if (module_instance == graph->inputs()[0]) {
           m = module;
         } else {
-          m = findChildModuleToQuantize(module, module_instance);
+          auto child_module_name = module_instance->node()->s(attr::name);
```

Comment:
this should happen after we check the kind of node is GetAttr actually

Reply:
@jerryzh168 yeah, updated it.